src/Pulp/Watch.purs

```diff
@@ -31,9 +32,14 @@ foreign import watch ::
   -> (String -> EffN Unit)
   -> EffN Unit
 
-watchAff :: Array String -> (String -> AffN Unit) -> AffN Unit
+-- launchAff's type in Control.Monad.Aff changed to forall e a. Aff e a -> Eff (err :: EXCEPTION) Unit
+-- which will cause a duplicate label in row error, here.  So use the old version for now.
+launchAff :: forall e a. Aff e a -> Eff e (Canceler e)
+launchAff = runAff (const (pure unit)) (const (pure unit)) 
+
```

Comment:
I think this code will swallow any exceptions thrown inside the inner Aff action, and I'd rather avoid this. What if you use the real `launchAff` but then define a specialisation of `unsafeCoerce` which strips away the `EXCEPTION` from the row?